share/doc/homebrew/Formula-Cookbook.md

```diff
@@ -303,14 +303,15 @@ correct. Add an explicit [`version`](http://www.rubydoc.info/github/Homebrew/bre
 
 ## Commit
 
-Everything is built on Git, so contribution is easy:
-
+Everything is built on Git, so contribution is easy:  
+1. Fork the Homebrew/homebrew-core repository on GitHub. This creates a pushable, personal remote repository. This is needed as only Homebrew maintainers have push access to the main repository.  
```

Comment:
Need a newline before this line. I think you don't need anything after `on GitHub.`